xsd/pmo/people.xsd

```diff
@@ -73,6 +73,18 @@ SOFTWARE.
           <xs:restriction base="xs:boolean"/>
         </xs:simpleType>
       </xs:element>
+      <xs:element name="skills">
+        <xs:complexType>
+          <xs:sequence>
+            <xs:element name="skill" minOccurs="0" maxOccurs="4" type="xs:string"/>
```

Comment:
@g4s8 I would also suggest to make it stricter than just `xs:string`. Let's add regex validation: `[a-z0-9+#.-]{2,16}`